OutOfSchool/OutOfSchool.WebApi.Tests/Extensions/MapSearchResultToOkOrNoContentExtensionTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using OutOfSchool.BusinessLogic.Models;
+using OutOfSchool.BusinessLogic.Models.Application;
+using OutOfSchool.WebApi.Extensions;
+
+namespace OutOfSchool.WebApi.Tests.Extensions;
+
+[TestFixture]
+public class MapSearchResultToOkOrNoContentExtensionTests
+{
+    private class TestController : ControllerBase
+    {
+    }
+
+    private TestController controller;
+
+    [SetUp]
+    public void SetUp()
+    {
+        controller = new TestController();
+    }
+
+    [Test]
+    public void MapSearchResultToOkOrNoContentExtension_SearchResultIsNotNullAndNotEmpty_ReturnOk()
+    {
+        // Arrange
+        var applications = new List<ApplicationDto>() {
+            new ApplicationDto() { RejectionMessage = "Test 1" },
+            new ApplicationDto() { RejectionMessage = "Test 2", },
+            new ApplicationDto() { RejectionMessage = "Test 3" },
+        };
+
+        var searchResult = new SearchResult<ApplicationDto>() { TotalAmount = applications.Count, Entities = applications };
```

Comment:
(Test data) YAGNI again. `SearchResult.TotalAmount` is not used in real code, thus could be not filled in testing code. Please drop setting of this property.